src/System.Text.Json/tests/JsonElementWriteTests.cs

```diff
@@ -437,6 +457,121 @@ public static void WriteNumberScientificAsProperty(bool indented)
                 "{\"m\\u00eal\\u00e9e\":1e6}");
         }
 
+        [Fact]
+        public static void WriteValueUnicodeEscapeSequence()
+        {
+            char[] chars = new char[] { (char)0x0421, (char)0x0430, (char)0x0439, (char)0x043D, ' ', (char)0x0443, (char)0x0443, '?', (char)0x04E9, (char)0x04E9 };
```

Comment:
Some of these lines are longer than github's default code view width (requiring horizontal scrolling).

Please break them up to improve the legibility of this code on github.

Reply:
ok